plugins/dependencies/src/main/resources/removals-revisions-history.md

```diff
@@ -262,3 +262,11 @@ id:[androidx.car:car]
 
 ~~AndroidX.tracingKtx~~
 id:[androidx.tracing:tracing-ktx]
+
+## [WIP]
+
+~~ReactiveX.rxJava2~~
+// The 2.x version is end-of-life as of February 28, 2021. No further development, support, maintenance, PRs and updates will happen.
+// TODO: Migrate to io.reactivex.rxjava3
+moved:[io.reactivex.rxjava3:rxjava3]
+id:[io.reactivex.rxjava2:rxjava2]
```

Comment:
This is intended for dependency notations that we are actually removing from refreshVersions. Here, you are adding it, so it should not be added here. Please, revert changes to this file.

Reply:
Gotcha. I _thought_ maybe that was what it was for, but it wasn't completely clear. I added these thinking, hey if you're using RxJava2 you should update to RxJava3